src/libraries/Microsoft.PowerFx.Core/Utils/StackMarker.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx.Core.Utils
+{
+    internal struct StackMarker
```

Comment:
>StackMarker [](http://example.com/codeflow?start=20&length=11)

Nit - should this be in Interpreter project?

Reply:
It should be in the same namespace as the exception. If neither class is used in this namespace, then both can be moved to the Interpreter project in the same namespace as EvalVisitor.

Reply:
I think this class ended up in this namespace because it was being used in the Interface (an earlier PR comment fixed that problem however, so this might be in the wrong place)

Reply:
Both assemblies contribute to the same Namespace, so we have freedom here.